examples/apps/presence-tracker/src/FocusTracker.ts

```diff
@@ -3,120 +3,74 @@
  * Licensed under the MIT License.
  */
 
-import { ISignaler } from "@fluid-experimental/data-objects";
 import { TypedEventEmitter } from "@fluid-internal/client-utils";
-import type { IAzureAudience } from "@fluidframework/azure-client";
-import { IContainer } from "@fluidframework/container-definitions/internal";
 import { IEvent } from "@fluidframework/core-interfaces";
-import { IMember } from "fluid-framework";
+import type {
+	IPresence,
+	ISessionClient,
+	LatestValueManager,
+	PresenceStates,
+} from "@fluidframework/presence/alpha";
+import { Latest, SessionClientStatus } from "@fluidframework/presence/alpha";
 
 export interface IFocusTrackerEvents extends IEvent {
-	(event: "focusChanged", listener: () => void): void;
+	(event: "focusChanged", listener: (focusState: IFocusState) => void): void;
 }
 
-export interface IFocusSignalPayload {
-	userId: string;
-	focus: boolean;
+export interface IFocusState {
+	hasFocus: boolean;
```

Comment:
Since this app is intended as an example, can we add comments to these? Also, can the properties be made `readonly`?